src/components/transactions/BatchExecuteButton/index.tsx

```diff
@@ -60,7 +60,7 @@ const BatchExecuteButton = () => {
             disabled={isDisabled}
             onClick={handleOpenModal}
           >
-            Execute batch{isBatchable && ` (${batchableTransactions.length})`}
+            Bulk execute{isBatchable && ` ${batchableTransactions.length} txs`}
```

Comment:
Not sure if the user knows what `txs` are. Maybe something like `Bulk execute (2)`?

Reply:
With `(2) txs` there's at least something to not know. With just `(2)` there's nothing to not know, so even less information.

Reply:
I would not use the term "txs" in the UI.

Reply:
OK, how about the full form?
<img width="1156" alt="Screenshot 2023-09-03 at 13 01 23" src="https://github.com/safe-global/safe-wallet-web/assets/381895/6f287749-4e3a-4789-96f3-6602be251da7">